src/app/campaign.model.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @link https://app.swaggerhub.com/apis/Noel/TBG-Campaigns/#/Campaign
  */
-export type Campaign = {
+export type CampaignOrMetaCampaign = {
```

Comment:
not exported, so code in other files has to distinguish whether it wants a charity campaign (aka simply a campaign) or a metacampaign.

We do have at least one routes that can technically be used for both, but I don't think in practice we've ever encouraged requesting a metacampaign by its id instead of it slug.